client/src/components/Chat/ChatMain.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from 'react';
+import DirectChatPage from './DirectChatPage.jsx';
+import axios from 'axios';
+
+const ChatMain = (props) => {
+  const [user, setUser] = useState('');
+  const [friends, setFriends] = useState('');
+
+  useEffect(() => {
+    axios.get('/user')
```

Comment:
Not too sure how the messaging works, but will you need a userid passed? The query call on the server for '/user' will need one to pull the user data, but I may be missing something here :)